examples/C#/README.md

```diff
@@ -0,0 +1,50 @@
+# Writing an SSE plugin using C#
+
+General background information is found here:
+
+* [Writing an SSE Plugin](../../docs/writing_a_plugin.md)
+* [Protocol Description](../../docs/SSE_Protocol.md) (API reference)
+* Documentation and tutorials for [gRPC](http://www.grpc.io/docs/) and [protobuf](https://developers.google.com/protocol-buffers/docs/overview)
+
+## Content
+* [Starting the server](#starting-the-server)
+* [RPC methods](#rpc-methods)
+    * [GetCapabilities method](#getcapabilities-method)
+    * [ExecuteFunction method](#executefunction-method)
+* ["Intelligent" functionality](#intelligent-functionality)
```

Comment:
Remove the documentation and code regarding "Intelligent" functionality. This will make the example easier to understand and will be in the category "Basic".